cogboard-webapp/src/components/WidgetTypeIcon.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import StatusIcon from './StatusIcon';
+import { StyledStatusIconButton, StyledIconWrapper } from './Widget/styled';
+
+const WidgetTypeIcon = props => {
+  const { type, content, status } = props;
+  return type === 'AemHealthcheckWidget' ? (
```

Comment:
Why only `AemHealthcheckWidget` have this feature? In my opinion it should be: 
`content.url ? StyledStatusIconButton : StyledIconWrapper`